client/utils.c

```diff
@@ -649,3 +649,36 @@ TDNFGetCmdOpt(
 error:
     return dwError;
 }
+
+uint32_t
+TDNFYesOrNo(
+    PTDNF_CMD_ARGS pArgs,
+    const char *pszQuestion,
+    int *pAnswer
+    )
+{
+    uint32_t dwError = 0;
+    int nAnswer = 0;
+    int32_t opt = 0;
+
+    if(!pArgs->nAssumeYes && !pArgs->nAssumeNo)
+    {
+        printf("%s ", pszQuestion);
+        while ((getchar()) != '\n');
+        opt = getchar();
```

Comment:
Do
```
opt = tolower(getchar());
```
Saves bunch of function calls over next lines

Also please call this method from:
tools/cli/lib/installcmd.c:TDNFCliAlterCommand() line 233